src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNavLinkItem.cs

```diff
@@ -58,23 +61,24 @@ public class BitNavLinkItem
         /// <summary>
         /// Whether or not the link is disabled
         /// </summary>
-        public bool Disabled { get; set; }
+        public bool IsEnabled { get; set; } = true;
 
         /// <summary>
         /// Link target, specifies how to open the link
         /// </summary>
         public string? Target { get; set; }
 
 
-                /// <summary>
+        /// <summary>
         /// A list of items to render as children of the current item
         /// </summary>
-        public List<BitNavLinkItem>? Links { get; set; }
+        public IEnumerable<BitNavLinkItem> Links { get; set; } = new List<BitNavLinkItem>();
 
         /// <summary>
         /// Callback invoked when a link in the navigation is clicked
         /// </summary>
-        public EventCallback? OnClick { get; set; }
+        public Action<BitNavLinkItem>? OnClick { get; set; }
+
         internal int Depth { get; set; }
```

Comment:
please add a definition comment for the `Depth` property.

Reply:
the access modifier of this property is `internal`, so it is not required for this prop to have documentation comment.